test/run

```diff
@@ -11,6 +11,8 @@ if [ -n "$TEST_SCENARIO" ]; then
     test/check-cli TestQcow2
   elif [ "$TEST_SCENARIO" == "tar" ]; then
     test/check-cli TestTar
+  elif [ "$TEST_SCENARIO" == "ci" ]; then
+	test/check-cli TestPylint 
```

Comment:
Why not call `make ci` (aka `make check test`) here like we discussed instead of starting a VM ?

Reply:
You also need the corresponding test scenario in `testmap.py` under `_manual` because we want to kick off this scenario and make sure it passes before merging this PR.

Reply:
Well I thought we want to run the make ci inside the VM which will have a specific fedora/rhel compose running...

Reply:
You are right. Scratch my previous comment.